### PR TITLE
Merge develop 9.6 16 7 2021

### PR DESCRIPTION
**Pull request by livecodepanos**
Conflict in `engine-sources.gypi`:

Screenshot 2021-07-16 at 12 30 04

Kept both changes.

### CODE REVIEW

File: engine/engine-sources.gypi [hunk 1]
```diff
@@ -23,17 +25,20 @@
...
 			'src/bitmapeffectblur.cpp',
 			'src/color.cpp',
 			'src/combiners.cpp',
+			'src/customfont.cpp',
 			'src/customprinter.cpp',
 			'src/font.cpp',
 			'src/fonttable.cpp',
 			'src/glcontext.cpp',
 			'src/glcontext.h',
+			'src/freetype-font.cpp',
```
Comment (livecodeian) [inline engine/engine-sources.gypi]: @livecodepanos could you place this before the glcontext lines to preserve alphabetical order

**Comment by livecodepanos** [comment]
@livecodeian Oops missed that! Thank you :)

**Comment by livecodeian** [comment]
@livecode-vulcan review ok 7a0292b

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeian ok 7a0292b34d041cbb861757a3f5a3cc28a0ba636e

**Comment by livecode-vulcan** [comment]
😎 test success 7a0292b34d041cbb861757a3f5a3cc28a0ba636e

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/1051)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/459)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/458)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/457)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/458)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2576)
* try-community-universal-ios-iphoneos14.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.5/builds/53)
* try-community-universal-ios-iphoneos14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.4/builds/105)
* try-community-universal-ios-iphonesimulator14.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.5/builds/53)
* try-community-universal-ios-iphonesimulator14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.4/builds/104)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2602)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2592)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2590)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2627)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2563)

**Pull request merged**